motoman_mh5_ikfast_manipulator_plugin/package.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version='1.0' encoding='ASCII'?>
+<package>
+  <name>motoman_mh5_ikfast_manipulator_plugin</name>
+  <version>0.0.0</version>
+  <description>The motoman_mh5_ikfast_manipulator_plugin package</description>
+  <!-- One maintainer tag required, multiple allowed, one person per tag -->
+  <!-- Example:  -->
+  <!-- <maintainer email="jane.doe@example.com">Jane Doe</maintainer> -->
+  <maintainer email="nitishsharma@todo.todo">nitishsharma</maintainer>
+  <!-- One license tag required, multiple allowed, one license per tag -->
+  <!-- Commonly used license strings: -->
+  <!--   BSD, MIT, Boost Software License, GPLv2, GPLv3, LGPLv2.1, LGPLv3 -->
+  <license>TODO</license>
```

Comment:
I know this came from the automatic gen tool but could you set the license to BSD and clean up the file a bit?